### DISCUSSION TITLE
Is it possible to add images to post list appearance?

### DISCUSSION THREAD

**Discussion by hiiiyilingzhang** [greenelab/lab-website-template#213] [Q&A] [answered]
Thanks so much for your development! I successfully build our website😃 

May I know if there're related features to add one image to post list? Something like [this](https://wowthemesnet.github.io/mundana-theme-jekyll/),
<img width="720" alt="截屏2023-10-13 16 24 55" src="https://github.com/greenelab/lab-website-template/assets/81615397/228c6f86-8727-4daf-8517-468573def93b">

I've tried [code here](https://talk.jekyllrb.com/t/how-to-grab-first-image-from-post-to-excerpt/2266) to extract images to post excerpt, but it seems like doesn't work for me. Could you please advise about how to achieve this feature? Many thanks!

Best,
Elaine

**Comment by vincerubinetti** [comment, marked as answer]
I think this is worthwhile as a built-in feature. I've created an issue here:

https://github.com/greenelab/lab-website-template/issues/214

I would provide a short-term immediate fix, but to implement this is somewhat complex with respect to CSS. MAYBE you could try this, but I haven't tested it. This is meant as a quick fix and not a permanent, robust solution.

In `_includes/post-excerpt.html`:

```liquid
<div class="post-excerpt">
  {% assign image = post.image %}
  <img src="{{ image | relative_url }}" style="height: 100px; float: left;" />

   ... etc...
</div>
```

Then in your post front matter, add an `image: images/some-photo.jpg` key.
